src/app/shared/components/delete-modal/delete-modal.component.html

```diff
@@ -1,19 +1,21 @@
 <ng-template #modal let-modal>
   <div class="modal-header">
-    <h5 class="modal-title" id="cloneModal">Are you sure you want to delete the following reports</h5>
+    <h5 class="modal-title" id="cloneModal">{{deleteQuestion}}</h5>
     <button type="button" class="close" (click)="modal.dismiss()">
       <span>&times;</span>
     </button>
   </div>
-  <div class="modal-body">
-    <ul>
-      @for (report of reports; track report) {
-        <li> [{{ report.storageId }}] {{ report.name }}</li>
-      }
-    </ul>
-  </div>
+  @if (!deleteAllReports) {
+    <div class="modal-body">
+      <ul>
+        @for (report of reports; track report) {
```

Comment:
Should be:
```suggestion
        @for (report of reports; track report.storageId) {
```

see [frontend conventions](https://github.com/frankframework/frontend-conventions/blob/master/angular.md#for-loops) for for-loops on arrays of objects.

Reply:
Added the change